assets/src/apps/v2/bus_shelter.tsx

```diff
@@ -34,7 +34,7 @@ import Placeholder from "Components/v2/placeholder";
 import LinkFooter from "Components/v2/bus_shelter/link_footer";
 import NormalHeader from "Components/v2/lcd/normal_header";
 import NormalDepartures from "Components/v2/departures/normal_departures";
-import SubwayStatus from "Components/v2/lcd/subway_status_v2";
+import SubwayStatus from "Components/v2/subway_status/lcd/subway_status";
```

Comment:
This organization change seems odd, but tbh our v2 components are all kinda disorganized. So I'mma ask about it in slack, so how we'd like things organized.

Reply:
The main reason for the new folder is so the common file has a sensible place to live. I couldn't think of an existing folder that would make sense for it, so I had it share a folder with the two components. Very much open to change on it.

Reply:
I guess I like things being organized by widgets, so I like there being a `subway_status` folder with a `common` file inside. At the moment, we don't plan on putting anymore styles in the lcd and e_ink folders, so maybe the css files could just be called `lcd_subway_status` and `eink_subway_status`

Reply:
You mean move the CSS files out of their eink/lcd folders and into a `subway_status` folder? That's definitely doable, but is a bit of a departure from the way the rest of our CSS is organized (not to say the current way is the best way).

Reply:
I think both the CSS and the TSX files could be called `lcd_subway_status.extension` and put right in the subway_status folder. But if that doesn't gel with our organization system, that's fine! It was a nitpick anyway

Reply:
I don't mind that for the components so I changed those names and moved the files. I think I'm gonna leave the CSS files how they are now to avoid a one-off for this component. I like the widget organization over app like how the components are organized, but it would be a larger task to get all CSS into that.